symbolic-symcache/src/new/mod.rs

```diff
@@ -1,45 +1,48 @@
 //! The SymCache binary format.
 //! # Structure of the format
 //!
-//! A SymCache contains the following primary kinds of data:
+//! A SymCache contains the following primary kinds of data, written in the following order:
 //!
-//! 1. address ranges
-//! 2. source locations
-//! 3. functions
-//! 4. files
+//! 1. files
+//! 2. functions
+//! 3. source locations
+//! 4. address ranges
+//! (5. string data)
```

Comment:
It's not really clear to me why this is parenthesized.

Reply:
It wasn't included originally, so I left it in parentheses in case this wasn't a first-class member of the list. It sounds like based on your reply that it's fine to strip the parentheses though, so I'll do that - do let me know if you'd rather see this removed instead.

Reply:
Leaving it in the list without parens is good.